genie-web/src/main/java/com/netflix/genie/web/agent/launchers/impl/LocalAgentLauncherImpl.java

```diff
@@ -48,11 +49,12 @@
     private static final String SETS_ID = "setsid";
     private static final String EXEC_COMMAND = "exec";
     private static final String SERVER_HOST_OPTION = "--serverHost";
-    private static final String SERVER_HOST_VALUE = "localhost";
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
+    private static final String SERVER_HOST_VALUE = "127.0.0.1";
     private static final String SERVER_PORT_OPTION = "--serverPort";
     private static final String API_JOB_OPTION = "--api-job";
     private static final String JOB_ID_OPTION = "--jobId";
-    private static final String FULL_CLEANUP_OPTION = "--full-cleanup";
+    private static final String FULL_CLEANUP_OPTION = "--no-cleanup";
```

Comment:
Variable name doesn't match value. Maybe a leftover from a test?

Reply:
Yeah I think this is a remnant of debugging but I want to leave it for now cause I do want `no-cleanup` so we can check the job directory etc as we do dev.

Eventually we probably want these to be some sort of properties but that's part of ongoing improvement